servatrice/src/servatrice_database_interface.cpp

```diff
@@ -283,6 +283,11 @@ ServerInfo_User Servatrice_DatabaseInterface::evalUserQueryResult(const QSqlQuer
 	const QString realName = query.value(3).toString();
 	if (!realName.isEmpty())
 		result.set_real_name(realName.toStdString());
+
+    const QString registrationDate = query.value(7).toString();
```

Comment:
I'm quite skeptical of storing dates in a char(20) field, since it will force us to use CAST() when we need to make a mathematical computation on the date in the db using sql.
I suppose we could achieve the same result using: 
```
const QString registrationDate = query.value(7).toDateTime().toString(Qt::ISODate);
```
As an alternative, the query could return the value of ``` DATEDIFF(NOW(), registrationDate) ``` as the number of days  in an integer field, avoiding any date calculation clientside an thus any timezone problem.